app/services/send_reject_by_default_email_to_provider.rb

```diff
@@ -14,4 +15,8 @@ def call
       ProviderMailer.application_rejected_by_default(provider_user, application_choice, can_make_decisions:).deliver_later
     end
   end
+
+  def do_not_send_notification?
+    CycleTimetable.after_end_of_cycle? && FeatureFlag.active?(:continuous_applications)
```

Comment:
Do we need the time check, given that we have the feature flag check? I assume the feature flag will only be active after the end of cycle anyways?

Reply:
the feature flag is active now – we have to use it in combination with the recruitment cycle year. We have a method on the `ApplicationForm` model `continuous_applications?` which covers this

Reply:
Ah yes. I did see that! 

I've updated it to use that method. Do we still want the `after_end_of_cycle?` check? I feel like the `CONTINUOUS_APPLICATIONS_CYCLE_YEAR` check means that will be handled automagically now, but want to double check.

Reply:
Yes i think we should just be fine with the CA check as this will ensure the app is in the 2024 cycle. Do you agree with this @tomas-stefano ?

Reply:
Continued the chat offline. @tomas-stefano has given his stamp of approval for removing the end of cycle check/method.